src/Components/ServiceScene/Breakdowns/PatternsViewTextSearch.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  CustomVariable,
+  SceneComponentProps,
+  sceneGraph,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneVariableSet,
+} from '@grafana/scenes';
+import React, { ChangeEvent } from 'react';
+import { Field, Icon, Input } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { PATTERNS_TEXT_FILTER } from '../../../services/variables';
+import { PatternsBreakdownScene } from './PatternsBreakdownScene';
+import { debouncedFuzzySearch } from '../../../services/search';
+
+export interface PatternsViewTextSearchState extends SceneObjectState {
+  patternFilter?: string;
```

Comment:
```suggestion
  patternFilter: string;
```

Default to empty string?

Reply:
fixed in https://github.com/grafana/explore-logs/pull/434/commits/7e1be70052a0d74901042bfdb4a3d81572a65ac6